.github/workflows/REVIEW_GUIDELINES.md

```diff
@@ -0,0 +1,712 @@
+
+// bad: http verbs not easily scannable
+Route::name('home')->get('/', 'HomeController@index');
+Route::middleware('openSource')->get('OpenSourceController@index');
+```
+
+Route parameters should use camelCase.
+
+```php
+Route::get('news/{newsItem}', 'NewsItemsController@index');
+```
+
+A route url should not start with `/` unless the url would be an empty string.
+
+```php
+// good
+Route::get('/', 'HomeController@index');
+Route::get('open-source', 'OpenSourceController@index');
+
+//bad
+Route::get('', 'HomeController@index');
+Route::get('/open-source', 'OpenSourceController@index');
+```
+
+## Controllers
+
+Controllers that control a resource must use the plural resource name.
+
+```php
+class PostsController
+{
+    // ...
+}
+```
+
+Try to keep controllers simple and stick to the default CRUD keywords (`index`, `create`, `store`, `show`, `edit`, `update`, `destroy`). Extract a new controller if you need other actions.
+
+In the following example, we could have `PostsController@favorite`, and `PostsController@unfavorite`, or we could extract it to a separate `FavoritePostsController`.
+
+```php
+class PostsController
+{
+    public function create()
+    {
+        // ...
+    }
+
+    // ...
+
+    public function favorite(Post $post)
+    {
+        request()->user()->favorites()->attach($post);
+
+        return response(null, 200);
+    }
+
+    public function unfavorite(Post $post)
+    {
+        request()->user()->favorites()->detach($post);
+
+        return response(null, 200);
+    }
+}
+```
+
+Here we fall back to default CRUD words, `store` and `destroy`.
+
+```php
+class PostsFavoriteController
+class UsersImageController
+class CompaniesLotController
+{
+    public function favorite(Post $post)
+    {
+        request()->user()->favorites()->attach($post);
+
+        return response(null, 200);
+    }
+
+    public function unfavorite(Post $post)
+    {
+        request()->user()->favorites()->detach($post);
+
+        return response(null, 200);
+    }
+    
+   public function mergeLot() {
+   
+   }
+}
+```
+
+This is a loose guideline that doesn't need to be enforced.
+
+## Views
+
+View files must use camelCase.
+
+```
+resources/
+  views/
+    openSource.blade.php
+```
+
+```php
+class OpenSourceController
+{
+    public function index() {
+        return view('openSource');
+    }
+}
+```
+
+## Validation
+
+All custom validation rules must use snake_case:
+
+```php
+Validator::extend('organisation_type', function ($attribute, $value) {
+    return OrganisationType::isValid($value);
+});
+```
+
+## Blade Templates
+
+Indent using four spaces.
+
+```html
+<a href="/open-source">
+    Open Source
+</a>
+```
+
+Don't add spaces after control structures.
+
+```html
+@if($condition)
+    Something
+@endif
+```
+
+## Authorization
+
+Policies must use camelCase.
+
+```php
+Gate::define('editPost', function ($user, $post) {
+    return $user->id == $post->user_id;
```

Comment:
Consider using strict comparison `===` instead of `==` for more predictable results.